src/OpenLoco/Objects/SteamObject.h

```diff
@@ -13,7 +13,11 @@ namespace OpenLoco
         uint8_t var_06;
         uint8_t var_07;
         uint16_t var_08;
-        uint8_t pad_0A[0x1E - 0xA];
+        uint32_t var_0A;
+        uint32_t var_0E;
```

Comment:
This is the `baseImageId` that all images will be relative for this object. (loaded at `0x440d5f`)